testdata/cv/io/Core_InputOutput--filestorage_base64_basic_read_JSON.json

```diff
@@ -45,6 +52,6 @@
         "rows": 8,
         "cols": 16,
         "dt": "d",
-        "data": "$base64$MWQgICAgICAgICAgICAgICAgICAgICAg/gyNZRq05j8igKa7xkXpP+T9CLHUQOo/gBWfIGNapT9Ez1DrK93sP3J5rLm0U+4/o+QNFKpQ5z/0F0cABL3jPxJAh2jqK94/+CT+oJEq7j/wy9sW7m+rP6DaEmm/p64/L78i3ang3z/s4vsqkjLBPxi9zkWQ2Mo/JQws1g0M2D+OWTZ2cYjdP7XV3VRPzO4/XGY511JnwT8QvNM0b9jhPwiiHKjl0KI/5qdQvi9f7j+K35ujLVDbPy7C/K0G8uY/0IXXmMu5kj92KiAvuW7cP3Q09XS+V8E/SMKdM+Totz/naRKE/bTlP8CI5WzaAe0/CDqZcAS90T9tO/ARNgPXP16sWoBTUeo/rgIU+fJn3z8zvr5CuyfiPzC4EMk93bA/Ro9xoAqX5j/Qjxm7ViXMP79WNhch5eg/sICi+G0byz9IFtYvZVDmP8p9GP7Nb8Q/nPAod4eD0T/XARjC9d/pP2iTyXR2ieE/jH0B02584j8paYJreSLnPyrr5qT5KuQ/qCqQvTPbzj/W+9sGdl/JP6eqNaEd/eY/342NCv1T3z8MaT1OUUbTP2jcuCyx5eA/vN5AEvZU6j+7CZTyT67fP1Hg/sWQOOk/FPGIWJ7E7D+jYayx/aLlP0t3hYmcke4/qyDO7RSH4D/+giCnrQHpP9u7CKBH3OM/tHxHvw7T3j8Cdj6z/r/gP1x0gUY1FNU/XLwZaBqZ5j+Avszkg7OQP4YFEGf2puk/Ilzpifst7T/PxyyJZ1ztP9P0DNPBLOA/zlZA5/wr0T/Krm973MTRPxCGYNOg78U/tqrp6XiA0T+3ALDjE/jaP+0u+Un4XtY/vFyaobKH7z+bHp6NVPnSP1LcYtxBwsQ/S4y0JL7D6T8LeMlnQr3uP/5K7nMCE+c/K9xmwgaA6j+gLkqCOU/pPxwO0uj7I98//HR67GlQyD/xaPX22OjtPwj6WtIGVNQ/Ll95Wa1R4D+A5gk4bASTP34XcDmj/eI/dOaigK495D84cxBlisnBP+DTE5urbOY/u3WMQwkp2z+ceeVRyAa5P46MMkB4ZOg/oo0gfLYHzz8AJ9yztvDKP7Yv2PQNYso/WOm1nVOl5T+PecWv8H7kPxZcBP0sw+Y/wI7GBq+k7D+wKsTmK2elP6AADZlYxq0/7lHRAq6G7z+iHF69FRrdPzOgfUs1juM/pXLjIA5X4j/UiUTSEc7QP26nXv0pccw/IWlWJrrk6j8cQSpExfvCPx0e1P8GR+w/2ecLCIqJ4T+ffYFSVaDjP8QhHBDfarA/aoERWDUw7z92um64hqzuP4l13HxN4ds/FxG6aQ7t3z/o9fcxzXO3P9huSYVv8sA/dD/n0FDh3j/sfJ70StbtPw=="
```

Comment:
I propose to put bool test after all existing cases. You do not need to modify existing test sequences then, just add the new one.